stream-video-android-core/src/main/kotlin/io/getstream/video/android/core/ParticipantState.kt

```diff
@@ -221,7 +221,13 @@ public data class ParticipantState(
         // _dominantSpeaker.value = participant.is_dominant_speaker. we ignore this and only handle the event
         updateAudioLevel(participant.audio_level)
         _audioEnabled.value = participant.published_tracks.contains(TrackType.TRACK_TYPE_AUDIO)
-        _videoEnabled.value = participant.published_tracks.contains(TrackType.TRACK_TYPE_VIDEO)
+
+        val hasVideoTrack = participant.published_tracks.contains(TrackType.TRACK_TYPE_VIDEO)
+        _videoEnabled.value = call.session?.trackOverridesHandler?.applyOverrides(
```

Comment:
Lets not update the `videoEnabled` for the participant, this is actually related to the remote participant state. We need to rely on the track overrides to hide the video if we disable it.

Reply:
This was one of the points that I raised as a red flag and, after discussing it, I'll check if the video is enabled directly in the UI.